guide/src/function/signature.md

```diff
@@ -188,3 +188,136 @@ impl MyClass {
     }
 }
 ```
+
+## Making the function signature available to Python
+
+The function signature is exposed to Python via the `__text_signature__` attribute. PyO3 automatically generates this for every `#[pyfunction]` and all `#[pymethods]`.
```

Comment:
I think this should mention that `text_signature` is generated based on the `signature` attribute optionally attached to the `#[pyfunction]`s and `#[pymethod]`s. Without that, one could expect this to be derived from the Rust function signature.

Reply:
Thanks, done